ZelBack/src/services/dockerService.js

```diff
@@ -987,60 +980,62 @@ async function dockerGetUsage() {
 
 /**
  * Fix docker logs.
+ * @returns {Promise<void>}
  */
 async function dockerLogsFix() {
-  try {
-    const nodedpath = path.join(__dirname, '../../../helpers');
-    const exec = `cd ${nodedpath} && bash dockerLogsFix.sh`;
-    const cmdAsync = util.promisify(nodecmd.get);
-    const cmdres = await cmdAsync(exec);
-    log.info(cmdres);
-  } catch (error) {
-    log.error(error);
-  }
+  const cwd = path.join(__dirname, '../../../helpers');
+  const scriptPath = path.join(cwd, 'dockerLogsFix.sh');
+  const { stdout } = await serviceHelper.runCommand(scriptPath, { cwd });
+
+  // we do this so we don't log empty lines if there is no output
+  const lines = stdout.split('\n');
+  // this always has length
+  if (lines.slice(-1)[0] === '') lines.pop();
+
+  lines.forEach((line) => log.info(line));
```

Comment:
Shouldn't this all continue inside a try catch?

Reply:
Yeah nice spot, the stdout.split() could throw, I'll fix.

Reply:
This highlighted an error I had in runCommand, stdout and stderr as allways returned as strings from child_process execFile. I've fixed this up (and tests) now that above code will never throw. Howver, for safety, have wrapped in a try / catch